public/js/comment.js

```diff
@@ -0,0 +1,26 @@
+const createCommentHandler = async (event) => {
+  event.preventDefault();
+
+  //   Eingabefelder habe ich so ausgelesen
+  const comment = document.getElementById("input-comment").value;
+  const postId = document.getElementById("post-id").textContent;
+  //   Values werden an den Server/API geschickt.
+  //   Mit Fetch wird der Pfad bestimmt, der über api_routes aufgerufen wird; in diesem Fall /post.
+  const response = await fetch("/api/comment", {
+    method: "POST",
+    // Pack die Values in den Body rein.
+    body: JSON.stringify({ comment, postId }),
+    headers: { "Content-Type": "application/json" },
+  });
+  if (response.ok) {
+    document.location.replace("/show-post-comment/" + postId);
```

Comment:
it may be better for the user experience to load all of the comments straight away on the page. This means you would only have to refresh the page on this line :-)